### PR TITLE
Update index.d.ts fixes #3549

### LINKED ISSUE #3549: Property 'after' does not exist on type 'ColumnBuilder'
# Environment

Knex version: ^0.20.2
Database + version: MySQL
OS: macOS 10.14.6

_@lorefnon / typescript_

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do

Should be able to use after() on ColumnBuilder

2. Error message

`Property 'after' does not exist on type 'ColumnBuilder'. Did you mean 'alter'?ts(2551)`

3. Reduced test code, for example in https://npm.runkit.com/knex or if it needs real
   database connection to MySQL or PostgreSQL, then single file example which initializes
   needed data and demonstrates the problem.

```js
knex.schema.alterTable('users', (t: Knex.TableBuilder) => {
      t.integer('creator_id')
        .after('source') // <== No worky
        .unsigned()
      t.foreign('creator_id').references('users.id')
    })
```

### PR DESCRIPTION
**Pull request by nvitaterna**
add after to columnbuilder definition

### CODE REVIEW

**Comment by kibertoad** [comment]
Thanks! Can you also add `first`? It also seems to be missing.

File: types/index.d.ts [hunk 1]
```diff
@@ -1808,6 +1808,7 @@ declare namespace Knex {
     alter(): ColumnBuilder;
     queryContext(context: any): ColumnBuilder;
     withKeyName(keyName: string): ColumnBuilder;
+    after(keyName: string): ColumnBuilder;
```
Comment (kibertoad) [inline types/index.d.ts:1811]: is `keyName` accurate here? I would imagine `field` or `column` would be more accurate.

Comment (nvitaterna) [reply types/index.d.ts:1811]: `columnName` would actually make sense, and is used slightly above in the `references` function.

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
Thanks a lot!

**Pull request merged**